source/agora/api/FullNode.d

```diff
@@ -144,6 +144,24 @@ public interface API
 
     public const(Block)[] getBlocksFrom (ulong block_height, uint max_blocks);
 
+    /***************************************************************************
+
+        Expose blocks as a REST collection
+
+        API:
+            GET /blocks/:height
+
+        Params:
+            _height = The height of the block to return
+
+        Returns:
+            The block at height `_height`, or throw an `Exception` (404).
+
+    ***************************************************************************/
+
+    @path("/blocks/:height")
```

Comment:
Doesn't it need to be `/block/:height`?

Reply:
I think he's trying to re-use the same end-point with an optional parameter.

Reply:
Yep, also there's a convention in REST interfaces that collections contain an `s`.